openedx_learning/apps/authoring/collections/models.py

```diff
@@ -44,11 +53,33 @@ class Collection(models.Model):
             "Whether the collection is enabled or not."
         ),
     )
-    created = models.DateTimeField(auto_now_add=True)
-    modified = models.DateTimeField(auto_now=True)
+
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+    )
+
+    created = models.DateTimeField(
+        auto_now_add=True,
+        validators=[
+            validate_utc_datetime,
+        ],
+    )
+
+    modified = models.DateTimeField(
+        auto_now=True,
+        validators=[
+            validate_utc_datetime,
+        ],
+    )
 
     class Meta:
         verbose_name_plural = "Collections"
+        indexes = [
+            models.Index(fields=["learning_package_id", "title"]),
+        ]
```

Comment:
I just wanted to confirm, since the ForeignKey field defined is `learning_package`, should this index be `learning_package` instead of `learning_package_id`? I was looking at other places in the code where indexes are defined, and the ForeignKey fields are defined without the `_id`, so I just wanted to make sure if I am missing something.

Reply:
I think you are right @yusuf-musleh !
Fixed here: 9417609aa58c94f5226ec8bd3722148d4c5d0445